src/components/PostsList.tsx

```diff
@@ -3,52 +3,68 @@
 import classNames from 'classnames';
 import React from 'react';
 import { Post } from '../types/Post';
-
+import {} from 'react-redux';
```

Comment:
The import statement for `react-redux` is empty. Ensure that any necessary hooks or components from `react-redux` are imported, or remove this line if it's not needed.